browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchActivity.java

```diff
@@ -13,15 +16,19 @@
  */
 public class BrowserSwitchActivity extends Activity {
 
-    private static Uri sReturnUri;
+    private static final String PREF_RETURN_URI = "pref_return_uri";
```

Comment:
Should put this preference in a separate file, and also provide a more descriptive key name (perhaps prefixed with `braintree_browser_switch`)

Reply:
Fixed